app/models/property.rb

```diff
@@ -4,6 +4,10 @@ class Property < ApplicationRecord
   attribute :value, :pence
   attribute :outstanding_mortgage, :pence
 
+  store_accessor :address, :address_line_one, :address_line_two, :city, :country, :postcode
+
+  OPTIONAL_ADDRESS_ATTRIBUTES = %w[address_line_two].freeze
```

Comment:
There is `Address` DB table/model - is that inappropriate for property address?

Reply:
> There is `Address` DB table/model - is that inappropriate for property address?

During [Spike](https://dsdmoj.atlassian.net/wiki/spaces/CRIMAP/pages/4697555085/Capital+Land+journey+backend+spike) for ‘Land and Property’ discussion, we decided not to use Address table and make it polymorphic. We decided to use properties.address(json) column instead